services/QuillLMS/client/app/bundles/Teacher/components/accounts/AuthGoogleAccessForm.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react'
+import getAuthToken from '../modules/get_auth_token'
+
+// TODO: These two constants should be populated via config/initializers/omniauth.rb
+export const AUTH_GOOGLE_ONLINE_ACCESS_PATH = "/auth/google/online_access"
+export const AUTH_GOOGLE_OFFLINE_ACCESS_PATH = "/auth/google/offline_access"
+
+const AuthGoogleAccessForm = ({
+  buttonClass = '',
+  formClass = '',
+  offlineAccess = false,
+  showIcon = true,
+  spanClass = '',
+  text
```

Comment:
Nice use of default parameters for default props.

Reply:
@emilia-friedberg I meant to ask you about this.  I chose empty string but wondered if `null` makes more sense here?